contracts/SmartLoansFactory.sol

```diff
@@ -55,10 +60,9 @@ contract SmartLoansFactory is IBorrowersRegistry {
     return smartLoan;
   }
 
-  function createAndFundLoan(uint256 _initialDebt) external payable returns(SmartLoan) {
+  function createAndFundLoan(uint256 _initialDebt) external oneLoanPerOwner payable returns(SmartLoan) {
     BeaconProxy beaconProxy = new BeaconProxy(payable(address(upgradeableBeacon)), abi.encodeWithSelector(SmartLoan.initialize.selector, address(priceProvider), address(assetsExchange), address(pool)));
     SmartLoan smartLoan = SmartLoan(payable(address(beaconProxy)));
-    smartLoan.initialize(priceProvider, assetsExchange, pool);
 
     //Update registry and emit event
     updateRegistry(smartLoan);
```

Comment:
I'm curious if it works as we don't transfer ownership as it happens with traditional createLoan. Is it verified in passing tests?

Reply:
I added a testcase for creating a loan using the `createAndFundLoan` method as it was not covered before, but I don't fully understand your comment as in line `75` of `createAndFundLoan` we do transfer the ownership just like in line `58` of the `createLoan()`